CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.3.2] - Dec 27, 2022
+
+- Added option to choose your own background color (Default is still white 😄).
+
```

Comment:
You can remove this, I'll do the notes and date when I publish the releases

Reply:
ok